plugins/modules/nmcli.py

```diff
@@ -411,6 +411,13 @@
         type: str
         choices: [ same_all, by_active, only_active ]
         version_added: 3.4.0
+    runner_fast_rate:
+        description:
+            - Option specifies the rate at which our link partner is asked to transmit LACPDU
+              packets. If this is true then packets will be sent once per second. Otherwise they
+              will be sent every 30 seconds.
+        type: bool
+        default: false
```

Comment:
Also I would remove the `default`, since having a default would be a breaking change. (If someone would have already set that value to `true` manually and would now use the new version of the module to modify the connection, the module would suddenly set it back to `false`.)

Reply:
Done